src/libraries/System.DirectoryServices.Protocols/tests/BerConverterTests.cs

```diff
@@ -30,8 +30,8 @@ public static IEnumerable<object[]> Encode_TestData()
             yield return new object[] { "[]", new object[] { "a" }, (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) ? new byte[] { 49, 132, 0, 0, 0, 0 } : new byte[] { 49, 0 } };
             yield return new object[] { "n", new object[] { "a" }, new byte[] { 5, 0 } };
 
-            yield return new object[] { "tetie", new object[] { -1, 0, 1, 2, 3 }, (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) ? new byte[] { 255, 1, 0, 1, 1, 2, 10, 1, 3 } : new byte[] { 10, 1, 0, 1, 1, 2, 10, 1, 3 } };
-            yield return new object[] { "{tetie}", new object[] { -1, 0, 1, 2, 3 }, (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) ? new byte[] { 48, 132, 0, 0, 0, 9, 255, 1, 0, 1, 1, 2, 10, 1, 3 } : new byte[] { 48, 9, 10, 1, 0, 1, 1, 2, 10, 1, 3 } };
+            yield return new object[] { "tetie", new object[] { 128, 0, 133, 2, 3 }, new byte[] { 128, 1, 0, 133, 1, 2, 10, 1, 3 } };
+            yield return new object[] { "{tetie}", new object[] { 128, 0, 133, 2, 3 }, (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) ? new byte[] { 48, 132, 0, 0, 0, 9, 128, 1, 0, 133, 1, 2, 10, 1, 3 } : new byte[] { 48, 9, 128, 1, 0, 133, 1, 2, 10, 1, 3 } };
```

Comment:
Nit: to be consistent we could use PlatformDetecion.IsWindows

Reply:
That is what I initially had on my original PR, but @bartonjs suggested to use RuntimeInformation.IsOSPlatform instead (which is what PlatformDetection is doing anyways). I'm happy to change it either way, don't really have a preference, although this whole test assmbly is using RuntimeInformation so if we want to change it I'll  have to do it for the rest of the test project.

Reply:
Ok, leave as is for the reason that all the test assembly uses it that way, but I believe most of the test assemblies use PlatformDetection, either way is fine.